config/software/m4.rb

```diff
@@ -31,6 +31,8 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
+  patch source: "m4-1.4.18-glibc-change-work-around.patch", plevel: 1, env: env if version == "1.4.18"
```

Comment:
is that patch appropriate for everything?  (rhel6, ubuntu, solaris, freebsd, etc?) or should it only be applied to rhel8?

Reply:
Good question. Let me see and find out. I think it's dependent the OS glibc version on the system.

Reply:
I have confirmed this patch works on the following platforms:
- CentOS 6, 7, 8
- Debian 8, 9, 10
- Ubuntu 16.04, 18.04
- FreeBSD 10, 11

I'm working on testing Solaris.